tests/sentry/event_manager/test_normalization.py

```diff
@@ -211,7 +196,7 @@ def test_logger():
     manager.normalize()
     data = manager.get_data()
     assert data['logger'] == DEFAULT_LOGGER_NAME
-    assert not any(e.get('name') == 'logger' for e in data['errors'])
+    assert not any(e.get('name') == 'logger' for e in data.get('errors', []))
```

Comment:
Should this not be `data.get('errors') or ()`?

Reply:
Sure, would be more robust but this is a testcase